pkg/probes/curl/curl_json.go

```diff
@@ -151,33 +150,20 @@ func (clp Probe) GetExpandedUserData(userDataVariables map[string]string) (strin
 		userDataVariables["CACERT_RENDERED"] = strings.TrimSpace(string(cloudInitYamlBytes))
 	}
 
-	// Also for compatibility reasons, we map the NOTLS variable to curl's "insecure" flag
-	if userDataVariables["NOTLS"] != "" {
-		noTLS, err := strconv.ParseBool(userDataVariables["NOTLS"])
-		if err != nil {
-			return "", fmt.Errorf("invalid userdata variable NOTLS: %w", err)
-		}
-		if noTLS {
-			userDataVariables["CURLOPT"] += " -k "
-			// In addition to adding the curl flag, we can merge the list of "tlsDisabled" URLs
-			// with the list of "normal" URLs now (since all URLs will be "tlsDisabled")
-			userDataVariables["URLS"] += " " + userDataVariables["TLSDISABLED_URLS"]
-			userDataVariables["TLSDISABLED_URLS"] = ""
-		}
+	curlCommand := curlgen.Options{
+		CaPath:          "/etc/pki/tls/certs/",
+		ProxyCaPath:     "/etc/pki/tls/certs/",
+		Retry:           3,
+		MaxTime:         userDataVariables["TIMEOUT"],
+		WriteOut:        fmt.Sprintf("%%{stderr}%s%%{json}\\n", outputLinePrefix),
```

Comment:
What's the motivation for removing `LINE_PREFIX` from the userdata and doing this here?

Reply:
But we don't want the `"@NV@"` to be configurable, right? That is not about stderr or stdout, but the characters we use to capture when the output should be parsed.